src/main/scala/org/apache/spark/sql/delta/isolationLevels.scala

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
should this file be called IsolationLevel.scala?

Reply:
the recommended style from the Spark world is that if there are multiple related top-level classes (that is, public classes used outside the file) are present in the same file, then it should be named not by any of the class but as common name across them starting with small letter. Example - https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/joinTypes.scala

Reply:
Ha, me and Andreas were just talking about this the other day. I thought that if there's one main one or it's an abstract class/trait and classes that extend it, it's named after the main one. But I guess this one could fit into either case. Either way fine with me